MapboxNavigation/NavigationViewController.swift

```diff
@@ -432,14 +399,11 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
     }
     
     func progressDidChange(notification: NSNotification) {
-        resetETATimer()
```

Comment:
Can we add this timer functionality back? It's there for when the user starts a navigation session and then does not move for some time.

Reply:
The timer just moved to `RouteMapViewController`. Should be the exact same functionality.